test/adding_tests.md

```diff
@@ -192,38 +205,43 @@ err := test.CheckConfigurationState(clients.ServingClient, configName, func(c *v
 })
 ```
 
-_See [crd_checks.go](./crd_checks.go) and
+_v1alpha1testing is alias for package `knative.dev/serving/pkg/testing/v1alpha1`_
+
+_For knative crd state, for example `Config`. You can see the code in [configuration.go](./v1alpha1/configuration.go).
+For kubernetes objects see
 [kube_checks.go](https://github.com/knative/pkg/blob/master/test/kube_checks.go)._
 
 ### Verify resource state transitions
 
 To use the [check functions](#check-knative-serving-resources) you must provide
 a function to check the state. Some of the expected transition states (as
-defined in [the Knative Serving spec](../docs/spec/spec.md)) are expressed in
-functions in [states.go](./states.go).
+defined in [the Knative Serving spec](https://github.com/knative/docs/blob/master/docs/serving/spec/knative-api-specification-1.0.md))
+, for example `v1alpha1/Revision` state, are expressed in
+function in [revision.go](./v1alpha1/revision.go).
 
 For example when a `Revision` has been created, the system will start the
 resources required to actually serve it, and then the `Revision` object will be
-updated to indicate it is ready. This can be polled with `test.IsRevisionReady`:
+updated to indicate it is ready. This can be polled with `v1alpha1testing.IsRevisionReady`:
 
 ```go
-err := test.WaitForRevisionState(clients.ServingClient, revisionName, test.IsRevisionReady(revisionName))
+err := v1alpha1testing.WaitForRevisionState(clients.ServingClient, revisionName, v1alpha1testing.IsRevisionReady(revisionName))
 if err != nil {
     t.Fatalf("Revision %s did not become ready to serve traffic: %v", revisionName, err)
 }
 ```
+_v1alpha1testing is alias for package `knative.dev/serving/pkg/testing/v1alpha1`_
 
 Once the `Revision` is created, all traffic for a `Route` should be routed to
-it. This can be polled with `test.AllRouteTrafficAtRevision`:
+it. This can be polled with `v1alpha1testing.AllRouteTrafficAtRevision`:
 
 ```go
-err = test.WaitForRouteState(clients.ServingClient, routeName, test.AllRouteTrafficAtRevision(routeName, revisionName))
+err := v1alpha1testing.WaitForRouteState(clients.ServingClient, routeName, v1alpha1testing.AllRouteTrafficAtRevision(routeName, revisionName))
```

Comment:
This needs to be updated because `AllRouteTrafficAtRevision` takes type `ResourceNames` as the argument.

Reply:
done